modules/ecs/locals.tf

```
@@ -1,3 +1,3 @@
 locals {
-  namespace = "ceramic-${var.env}-${var.base_namespace}"
+  namespace = "ceramic-elp-${var.base_namespace}"
```

No more ELP 🙌 (now that we are on v2)
```suggestion
  namespace = "ceramic-${var.env}-${var.base_namespace}"
```

---

Lol, yes. We'll have to finagle this to get it to work for the migration though. The `ceramic_env` was set to `elp` so it matched the infra we have now, but using `env` will map everything to `ceramic-prod*` and not `ceramic-elp*`.

---

I can still pass it in as config, just not call it `ceramic_env`.

---

ohhhh ok ok. i remember now why I named it that. I guess ceramic_env is better actually